drivers/hmis/app/models/hmis/hud/user.rb

```diff
@@ -21,11 +21,13 @@ class Hmis::Hud::User < Hmis::Hud::Base
   has_many :custom_data_elements, **hmis_relation(:UserID, 'CustomDataElement')
   belongs_to :data_source, class_name: 'GrdaWarehouse::DataSource'
 
+  # Find or create the Hmis::Hud::User corresponding to the provided application user (Hmis::User)
   def self.from_user(user)
-    Hmis::Hud::User.where(user_id: user.id, data_source_id: user.hmis_data_source_id).first_or_create do |u|
-      u.user_email = user.email
+    Hmis::Hud::User.where(user_email: user.email, data_source_id: user.hmis_data_source_id).first_or_create do |u|
```

Comment:
I think we should downcase to prevent some duplicates
```suggestion
    Hmis::Hud::User.where(user_email: user.email.downcase, data_source_id: user.hmis_data_source_id).first_or_create do |u|
```

Reply:
`email` on the user should always be lowercase, I think. But will update.